app/views/articles/_comment_form.html.erb

```diff
@@ -24,6 +24,10 @@
       <div class="comment-form__group"><%= raw recaptcha_tags ajax: true %></div>
     <% end %>
 
+    <div class="comment-form__group comment-form__group--policy">
+      <p><%= t(".by_clicking_on") %><a href="/pages/commenting-policy"><%= t(".commenting_policy") %></a></p>
+    </div>
+
     <div class="comment-form__group comment-form__group--submit">
       <%= link_to t('.cancel'), '#' %> <input class="button button--primary comment-form__submit-button" type="submit" value="<%= t('.post_comment') %>">
```

Comment:
What will the hidden link do without Javascript? If nothing, we should hide it for non-js